osm-fudge/__main__.py

```diff
@@ -0,0 +1,27 @@
+import bk_tree
+import sys
+
+import levenshtein
+import jaccard
```

Comment:
At the moment we have global modules. What we probably should do is create a proper project specific module and then have these in sub-modules. Here is an example how this could look like:

```
osm-fudge/
    mymodule/
        __init__.py
        metrics/
            __init__.py
        ..
```

Then on the call site you would do e.g. `import mymodule.metrics` or `from mymodule.metrics import jaccard`.